src/app/shared/checkMobile.service.ts

```diff
@@ -16,6 +19,11 @@ export class CheckMobileService {
 
   checkIsMobile(): boolean {
     this.setScreenWidth();
-    return this.getScreenWidth() <= 780 ? true : false;
+    return this.getScreenWidth() <= mobileBreakpoint ? true : false;
+  }
+
+  checkIsTablet(): boolean {
+    this.setScreenWidth();
+    return this.getScreenWidth() <= tabletBreakpoint ? true : false;
```

Comment:
not sure about the implication but instead of checking two different case we can use one function to check the type. As this function will also return true for mobile size if checked against table.

Reply:
That's true. 

In a unified function to check the device type, what other device info would you look out for, orientation?